helpers/atelierAPI.js

```diff
@@ -0,0 +1,41 @@
+const axios = require('axios');
+const config = require('../config.js');
+
+const host = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp';
+const headers = {
+  'User-Agent': 'request',
+  'Authorization': `token ${config.TOKEN}`
```

Comment:
you dont need the word token for the authorization, just the token itself

Reply:
Changed.